opentelemetry-sdk/src/logs/record.rs

```diff
@@ -1,10 +1,20 @@
+use super::growable_array::GrowableArray;
 use opentelemetry::{
     logs::{AnyValue, Severity},
     trace::{SpanContext, SpanId, TraceFlags, TraceId},
     Key,
 };
 use std::{borrow::Cow, time::SystemTime};
 
+// According to a Go-specific study mentioned on https://go.dev/blog/slog,
+// up to 5 attributes is the most common case. We have chosen 8 as the default
```

Comment:
i am inclined to stick with 5 itself.

Reply:
I should be fine with either of them. Will make it 5.